src/main/java/com/github/checkstyle/regression/Main.java

```diff
@@ -19,11 +19,45 @@
 
 package com.github.checkstyle.regression;
 
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+import com.github.checkstyle.regression.configuration.ConfigGenerator;
+import com.github.checkstyle.regression.data.GitChange;
+import com.github.checkstyle.regression.data.ModuleExtractInfo;
+import com.github.checkstyle.regression.data.ModuleInfo;
+import com.github.checkstyle.regression.extract.ExtractInfoProcessor;
+import com.github.checkstyle.regression.git.DiffParser;
+import com.github.checkstyle.regression.module.ModuleCollector;
+import com.github.checkstyle.regression.module.ModuleUtils;
+import com.github.checkstyle.regression.report.ReportGenerator;
+
 /**
  * Utility class, contains main function and its auxiliary routines.
  * @author LuoLiangchen
  */
 public final class Main {
+    /** Option name of repository path. */
+    private static final String OPT_REPO = "repo";
```

Comment:
I made this changes:
`repo` => `localGitRepo`, and short opt is `r`
`branch` => `patchBranch` and short opt is `p`

Do we really need a parameter of `baseBranch`, is there any possible that it is not `master`?

The option name of checkstyle-tester path is being discussed in the post below.

Short opts are defined in `createOptions` directly. Need I create static final field instead of writing them in plain text in `createOptions`?